test/test-pages/pixnet/expected-metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "新竹尖石_美樹營地賞楓 (2) @ 史蒂文的家_藍天 :: 痞客邦 PIXNET ::",
+  "byline": "史蒂文的家_藍天 (stevenhgm)",
+  "dir": null,
+  "excerpt": "一波波接續性低溫寒流報到 已將新竹尖石鄉後山一帶層層山巒披上嫣紅的彩衣 玉峰道路一路上雲氣山嵐滯留山頭 順路下切蜿蜒道路後不久即抵達來到&quot;玉峰國小&quot; &quot;美樹&quot;美",
```

Comment:
I don't speak the language used here (Chinese, I think?) - does the leading em-dash make sense to start the excerpt with?

I mean, this is good to land either way and if it's an issue we can address it in a separate patch.